### PR TITLE
Creating Fully Functional Docker Image

### LINKED ISSUE #18: Docker image needed
Hi! I'm want to run nocode with millions of containers on huge scale. And because now configuration and build processes are very hard to repeat on big number of nodes i think docker support will be very desired!

### PR DESCRIPTION
**Pull request by taiidani**
Fixes #18

### CODE REVIEW

**Comment by hypersleep** [comment]
Such a great work! Thank you so much @taiidani !

**Comment by Redmega** [comment]
This is code

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1 @@
+FROM scratch
```
Comment (drewcassidy) [inline Dockerfile]: Is code, please remove

**Comment by taiidani** [comment]
Closing due to Kelsey already making this change to `master` outside PR. Good times!

**Pull request closed without merging**